backend/src/middlewares/auth/strategies/google.strategy.ts

```diff
@@ -13,9 +13,8 @@ const googleStrategy = new GoogleStrategy(
     scope: ['profile', 'email'],
   },
   async (req, accessToken, refreshToken, profile, done) => {
-    let user;
     try {
-      user = await prisma.user.findFirst({
+      let user = await prisma.user.findFirst({
```

Comment:
Why not `const`? Will the `user` be reassigned?

Reply:
Yeah, it's reassigned in the code below

Reply:
Okay, I just saw it below :ok_hand: